repotracker/repotracker.go

```diff
@@ -581,9 +580,22 @@ func addBuildBreakSubscriptions(v *version.Version, projectRef *model.ProjectRef
 		return nil
 	}
 	subscriptionBase := event.Subscription{
```

Comment:
Can you make this a function in model/events?

Reply:
I don't think it belongs there - it's too specialized to be public